### DISCUSSION TITLE
post/martin-dev-honey-tip-1/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#11] [Announcements]
# post/martin-dev-honey-tip-1/

Json 공백 문자 제거 및 파싱을 개발하면서 Jackson Deserializer 코드를 분석해봅니다.

https://tech.kakaopay.com/post/martin-dev-honey-tip-1/

**Comment by ACACIANh** [comment]
좋은 글 감사합니다
